### DISCUSSION TITLE
firelfy linux : files used in the system / use only encrypted volume

### DISCUSSION THREAD

**Discussion by ehanuise** [iotaledger/firefly#1831] [Help & Support] [answered]
Hello everyone.
I have a question about the linux desktop wallet : which files does it create/use on the computer ?
For other wallets, I usually create an encrypted volume and configure the wallet to only use that volume, so I can be sure all files created/used remain on encrypted volumes. (By using the wallet configuration, or by setting up symlinks if the files locations cannot be changed easily)
I did not find any options/settings about this in firefly desktop.
I also looked in the documentation but couldn't find anything about filesystem files location/usage.
Any pointers to the documentation for this are welcome :-)

**Comment by ehanuise** [comment, marked as answer]
I found files in /home/user/.config/Firefly
Are there other files ?

**Reply by svenger87** [reply]
No. That's the only directory where FireFly stores config related files.

**Reply by ehanuise** [reply]
Thank you !
